api/search-tags.md

```diff
@@ -17,90 +17,9 @@ GET
 ### Inputs
 
  * `account_id`          : The account to search
- * `query`               : See below
+ * `query`               : See [Tag Queries](/tag_queries.md) for more info on tag queries.
  * `activity_start_secs` : (optional) The start time from which to pull data, represented in seconds since the unix epoch.
  * `activity_end_secs`   : (optional) The end time up to which data is pulled, represented in seconds since the unix epoch.
-  
-## Query syntax
```

Comment:
Why move tag syntax to another page?  It doesn't look like you've modified this section at all.

Reply:
it was embedded inside the page for find, but all the deletes need to refer to it.
so i had to choose whether to point all the delete pages at the find page where the tag info was embedded, or move it to a new page.
made more sense to make it a new page and make find and the deletes all point to it.

Reply:
You can link to a specific anchor within a page, FYI:

`[query syntax](/api/search-tags.md#query-syntax)`

I think there is value in keeping the syntax together with the search docs.  Splitting them up seems to regress the case where I want to learn about searching for tags in general.